### ISSUE TITLE
It doesn't do anything! :O

### ISSUE THREAD

**Issue by SalomonSmeke** [joehernandez11/Twitter_Project#2]
Hm. You might need to keep working on this. I do understand the software engineering approach of starting by creating your class containers and all that and THEN doing things.

But you gotta have something.

Currently. This is an equivalent application in scala, except this version handles multiple tweets:

val out = scala.io.Source.stdin.getLines.toList.map(x => x.split" ");
out.map(x.map(println (x)));

**Comment by jthompson18** [comment]
@joehernandez11 on a similar note to @SalomonSmeke's comment. I can see where you want to go with this, but you appear to be lacking on the execution side of things. If you have questions don't hesitate to ask on slack or you can DM me as well. For future projects I recommend starting with writing down your algorithm in plain english, like so:
1. parse message
   1. extract mentions beginning with '@'
   2. extract topics beginning with '#'
   3. extract URLs

An exercise like this will help you get started on a given project. You can then work on implementing it in whatever language you'd like. The next step I'd recommend is determining how to approach the algorithm. Because messages are string based I'd recommend using Regular Expressions (regex). Feel free to check out my [implementation](https://github.com/jthompson18/COMP330/tree/master/messenger). Its in python which is fairly straight forward language. Feel free to reach out to me with any questions you might have.

**Comment by joehernandez11** [comment]
I understand its in the early stages. Thank you for taking the time to review what I have and to leave a comment. I also like the brainstorming tactic to "write it in plain english". It will surely help me to narrow what exactly needs to get done and how.
